setup/insertObjects.js

```diff
@@ -20,8 +85,8 @@ Kompus06_1 = [{
     "Inlopp": "O: 110, typ: gummitätning, VG: 850",
     "Kabelgenomföring": "O: 50, typ: gummitätning, VG: 1800",
     "Antal pumpar": 1,
-    "Pump": "GD-20  Skärande avloppspump, 3-fas",
-    "ArtikelNr": "K06-1GD20A",
+    "Pump": "GD-20",
+    "Modell": "K06-1GD20A",
```

Comment:
This should not be named Modell.... A Modell name is already set for Kompus 06_1